rolling-hash.md

```diff
@@ -1,11 +1,11 @@
 # Rolling Hash Algorithm
-_Spec v4 (2021-03-09)_
+_Spec v5 (2022-04-04)_
 
-Make a rolling hash based file diffing algorithm. When comparing original and an updated version of an input, it should return a description ("delta") which can be used to upgrade an original version of the file into the new file. The description contains the chunks which:
+Make a rolling hash based file diffing algorithm. When comparing original and an updated version of an input, it should return a description ("delta") which can be used to upgrade an original version of the file into the new file. The description provides information of the chunks which:
 - Can be reused from the original file
-- have been added or modified and thus would need to be synchronized
+- Have been added or modified and thus would need to be synchronized
 
-The real-world use case for this type of construct could be a distributed file storage system. This reduces the need for bandwidth and storage. If many people have the same file stored on Dropbox, for example, there's no need to upload it again.
+The real-world use case for this type of construct could be a distributed file storage system. This reduces the need for bandwidth and storage. If user has a local copy of a file stored in the cloud, then changes between these two instances can be synchronized using diff produced by rolling hash.
```

Comment:
Single cryptographic hash makes more sense when deciding whether two different users have the same file. Synchronizing changes between two distinct instances of the same file is better use case for rolling diff.